docs/system/mx4/reset_cause.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Reset cause (MX-4)
+tags:
+  - Reset cause
+  - MX-4
+---
+
+You can read out the platform "reset cause" from an SPI attribute or using the `/opt/hm/reset_cause.sh` script.
+Multiple reset causes are possible and you can reset the current reset cause by executing `echo 0 > /opt/hm/pic_attributes/ctrl_pic_reset_cause`.
```

Comment:
```suggestion
Multiple reset causes are possible. You can also zero out the current reset cause by executing `echo 0 > /opt/hm/pic_attributes/ctrl_pic_reset_cause`.
```

Reply:
I use "zero out" above only  for stylistic reasons. Otherwise you get a lot of "reset" in the same sentence.